gradle.properties

```diff
@@ -1,19 +1,19 @@
 org.gradle.parallel=true
 
-spinnakerGradleVersion=8.15.0
+spinnakerGradleVersion=8.32.1
 pf4jVersion=3.2.0
-korkVersion=7.123.0
+korkVersion=7.236.0
 # The service versions listed below are not needed for the actual plugin to function
 # but are being added in order to enable plugin compatibility testing for the plugin
-clouddriverVersion=2022.02.24.19.52.46.release-1.27.x
-echoVersion=2022.01.19.07.51.20.release-1.27.x
-fiatVersion=2022.03.02.18.55.09.release-1.27.x
-front50Version=2022.01.19.08.45.30.release-1.27.x
-gateVersion=2022.02.03.20.15.21.release-1.27.x
-igorVersion=2022.01.19.07.32.50.release-1.27.x
-kayentaVersion=2022.02.16.00.31.30.release-1.27.x
-orcaVersion=2022.02.15.22.24.23.release-1.27.x
-roscoVersion=2021.08.03.20.41.14.master
+clouddriverVersion=2024.07.17.17.30.47.master
```

Comment:
these versions are not the latest changes in master. Can we change it to latest?

Reply:
It was the newest at the time of the PR lol, will update this. Hopefully, it will work and we can merge it.

Reply:
This shouldn't matter THAT much.  Bigger thing is getting micrometer versions to match.  WHICH may have more ... impactful changes.